docs/specs/xcmp.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+title: XCMP Interface
+parent: Specification
+nav_order: 2
+---
+
+# XCMP Interface <!-- omit in toc -->
+
+_NOTE: work in progress_
+
+- [Introduction](#introduction)
+- [Scenarios](#scenarios)
+  - [Alice transfers 21 PolkaETH to Bob on another chain](#alice-transfers-21-polkaeth-to-bob-on-another-chain)
+  - [Alice attempts to exchange 42 DOT for 21 PolkaETH](#alice-attempts-to-exchange-42-dot-for-21-polkaeth)
+- [Asset Identification](#asset-identification)
+- [Outstanding Issues](#outstanding-issues)
+  - [Numeric precision](#numeric-precision)
+
+## Introduction
+
+Our parachain will support cross-parachain token transfers using [XCMP](https://github.com/paritytech/xcm-format/blob/master/README.md).
+
+Since our parachain endeavours to be trustless, only reserve-backed transfers will be supported. Participating chains will need to agree on a parent chain to hold sovereign (reserve) accounts.
```

Comment:
@vgeddes Parachains wanting to use our bridge need to trust us anyway, which implies that we can force them to use us as the effective chain to hold sovereign accounts. If this simplifies things then we should consider this design too instead of a design in which the relay chain holds the sovereign account. What do you think?

Reply:
Great idea, I like it, especially as it means we don't have to trust another reserve chain and manage sovereign accounts.